hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockDataStreamOutput.java

```diff
@@ -627,6 +644,8 @@ private void writeChunkToContainer(ByteBuffer buf)
                 CompletionException ce = new CompletionException(msg, e);
                 setIoException(ce);
                 throw ce;
+              } else if (r.isSuccess()){
+ // TODO       xceiverClient.updateCommitInfosMap(r.getCommitInfos());
```

Comment:
is something missing here?

Reply:
ratis 2.3-snapshot too doesn’t have this code